src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write code here
+const workersList = document.querySelector('ul');
+const workersArr = [...workersList.children];
+
+workersList.innerHTML = '';
+
+sortList(workersArr).forEach((worker) => workersList.append(worker));
+
+function getSalary(worker) {
+  const workerSalary = worker.getAttribute('data-salary').replaceAll('$', '');
```

Comment:
The `replaceAll` method is not supported in Internet Explorer. If compatibility with older browsers is required, consider using a regular expression with `replace` instead.